src/components/material/MaterialDescription.tsx

```diff
@@ -17,20 +17,30 @@ export interface MaterialDescriptionProps {
 const MaterialDescription: React.FC<MaterialDescriptionProps> = ({ work }) => {
   const t = useText();
   const { searchUrl, materialUrl } = useUrls();
-  const inSeries = work.series;
-  const seriesMembersList = work.seriesMembers.map((item) => {
+  const { fictionNonfiction, series, subjects, seriesMembers, manifestations } =
+    work;
+
+  const seriesMembersList = seriesMembers.map((item) => {
     return {
       url: constructMaterialUrl(materialUrl, item.workId as WorkId),
-      term: item.titles.main[0]
+      term: item.titles.main.join(" / ")
```

Comment:
What is this change about?

As far as I can tell it does not have any relation to the change at hand and I do not see any explanation of it in the commit comments either.